_posts/2018-08-28-socrates-germany-2018.md

```diff
@@ -0,0 +1,51 @@
+The paper is scary, but you need to not be afraid of making mistakes. Also don't judge your drawings.
+Frustration took over for me, I tilted. Thanks for your kind words [Yulia](https://twitter.com/ioctaptceb). I've learned that I need to be more compassionate of myself, accepting my failures, defects, limitations, and shortcomings.
+
+This is the kind of surprise I go to SoCraTes for. Mind flipped and ego shrunk I headed to the sauna.
+
+# Sunday
+
+I cannot share the content of the next session, the topic was Sexism. I hope by hearing examples of bad behaviours I can learn to recognise them, in myself and in others. I think hearing them told directly made me empathise.
+A simple suggestion, as an ally, was to compliment colleagues. I want to read more on this subject and [these](https://stackoverflow.blog/2017/10/04/3-ways-can-ally-women-tech/) seems like simple suggestions.
+
+[OWASP Juice shop](https://www.owasp.org/index.php/OWASP_Juice_Shop_Project) (one of the few technical sessions I went to). Run a node application on your laptop and try to hack into it. I think this was marvelously gamified, each hack awarding a new achievement in varying difficulty levels. Wow!
+
+The most important takeaway from the next session, working intercultural teams, is that culture has an influence in how we think about power distance, the need for rules, and the balance between individualism and collectivism, long and short term thinking, and "masculinity and femininity". These differ from person to person and do influence our ability to communicate and collaborate.
+One everyday example of this is that the perception of time is culturally mediate: "5 minutes" actually means something different depending on who are you talking with.
+
+Refuctoring was another "technical" session: start with a plain Fizz-Buzz implementation in JavaScript and turn it into the most [diabolically intricate piece of code](https://github.com/Hackerkegeln/refuctoring/blob/socratees2018/src/fizz-buzz-generator.js) we could. The insight for me happened when the code crossed my mental ability to model its behaviour, which happened upon inlining a function.
+Immediately after my brain just gave up, my mental model was gone, garbage collected. My mind was empty and it took me a good 10 minutes to understand that part of the code again. To recreate a mental model.
+Fascinating.
+
+There was more, we talked about the five languages of love, [played board games](https://twitter.com/coderbyheart/status/1033076918160162819), heard about the book Accelerate, discussed code reviews and continuous delivery, and collected a lot of laptop stickers.
+
+I had an amazing time and I'm so grateful to the organisers and everyone that come to SoCraTes Germany this year. [Franzi](https://twitter.com/Singsalad) in particular, for being a great travel buddy and facilitator.
+
+See you again in a year!
```

Comment:
Would you like to mention the workshops on Sun?

Reply:
I did not do any :)

Reply:
I remember :)